handlers/easy_info/__init__.py

```diff
@@ -0,0 +1,12 @@
+from eventhandler import EventHandler
+
+
+class EasyInfoHandler(EventHandler):
+    def on_issue_labeled(self, api, payload):
+        new_label = payload['label']['name']
+
+        if new_label == 'E-Easy':
```

Comment:
We've standardized on adding the same E-Easy label to all our repositories since we hardcoded it in other tools like https://github.com/servo/servo-starters . Configuration won't yield any benefit at the moment; it would mean a little less work in the future to generalize highfive for the benefit of other projects.